lib/ardb/adapter/base.rb

```diff
@@ -12,6 +12,8 @@ def initialize
     schema_path = Ardb.config.schema_path
     @ruby_schema_path = "#{schema_path}.rb"
     @sql_schema_path  = "#{schema_path}.sql"
+
+    @verbose = !(ENV["VERBOSE"] == "false")
```

Comment:
@Criptak - Can we namespace these with `MIGRATE_` so `MIGRATE_VERBOSE` in this case. Also move this down into the `migrate_db` method so its specifically for the `migrate_db` method.